pkg/rancher-desktop/main/networking/index.ts

```diff
@@ -65,15 +65,15 @@ export default async function setupNetworking() {
       pluginDevUrls.some(x => url.startsWith(x))
     ) {
       event.preventDefault();
-      // eslint-disable-next-line node/no-callback-literal
+      // eslint-disable-next-line node/no-callback-literal, n/no-callback-literal
```

Comment:
We are pulling in both `node/` and `n/` variants? From the [_standard_ changelog](https://github.com/standard/standard/blob/v17.0.0/CHANGELOG.md#major-changes) (that's a horrible package name), it sounds like `n/` is a fork of `node/`…

Reply:
Yeah, this is what I didn't like about my initial pass at this..

What I would really like to see is if we could target one or the other instead of having the two rules commingled. Unfortunately, I might have to shelve this for the time being in favor of feature work. That is, unless we have a simple fix in mind 🙂